tor_ocr/strings.py

```diff
@@ -4,5 +4,7 @@
     'At the very least, hopefully it will serve as a decent starting point '
     'for your work!\n\nPlease note that any formatting instructions above '
     'override whatever I provide, so please format my content accordingly '
-    'if you choose to use it.'
+    'if you choose to use it. '
+    '&nbsp; '
```

Comment:
Let's replace the nonbreaking space with "\n\n" as further up in the message -- that way we're 100% sure it will play nicely with desktop & mobile.

Reply:
I believe &nbsp; works on on mobile clients now, thanks to that awesome Reddit update, but yeah I'll fix that.

Reply:
It might with the official reddit client, but I, for one, prefer to use third party clients.

Definitely in favor of the version that's more compatible across the board. 👍